manifests/base/resources/qlikview-client/secrets/selectivepatch.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: qlik.com/v1
+kind: SelectivePatch
+metadata:
+  name: qlikview-client-component-secrets
+enabled: true
+patches:
+- target:
+    kind: Secret
+  patch: |-
+    - op: add
+      path: /stringData
+      value:
+        mongoDbUri: mongodb://$(PREFIX)-mongodb:27017/PDS
```

Comment:
`qlikview-client` doesn't have a mongo dependency. So we may need to rmv that plz.

Reply:
On it..